src/modules/app/app.controller.spec.ts

```diff
@@ -0,0 +1,23 @@
+import type { TestingModule } from '@nestjs/testing';
+import { Test } from '@nestjs/testing';
+import { AppController } from './app.controller';
+import { AppService } from './app.service';
```

Comment:
```suggestion
import { Test } from '@nestjs/testing';
import { AppController } from './app.controller';
import { AppService } from './app.service';

import type { TestingModule } from '@nestjs/testing';
```

старайся всюди переносити імпорти типів нижче